multi-agent/lib/mas/session/building/category_builders/node_builder.py

```
@@ -25,13 +28,27 @@ class NodeBuilder(CategoryBuilder):
+        """Resolve all Ref-typed fields and auth credentials."""
+        kwargs: Dict[str, Any] = {
             name: self._resolve(getattr(cfg, name, None), cfg, reg)
             for name in self._get_ref_field_names(cfg)
         }
 
+        server_id = getattr(cfg, "server_identifier", "")
+        scheme_type = getattr(cfg, "scheme_type", "")
+        if server_id and deps and deps.auth_service:
+            ctx_holder = getattr(deps, "execution_ctx", None)
+            if ctx_holder:
+                resolver = lambda _h=ctx_holder: _h.context.credential_user_id()
```

Ruff (0.15.20)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `resolver` as a `def`

(E731)